sbt-plugin/src/sbt-test/gen-scala-server/00-interop/src/main/resources/application.conf

```diff
@@ -0,0 +1 @@
+akka.http.server.preview.enable-http2 = on
```

Comment:
This scripted test was changed to use the normal `Http` API instead of the now-dropped `Http2` API, which is why it didn't need the setting before but does now.

I'll port this change to a PR on master and then rebase this branch, to keep things together that should be together :)

Reply:
#1177